src/components/modals/SelectTokenModal/SelectTokenModal.vue

```diff
@@ -220,6 +225,23 @@ export default defineComponent({
       ...(props.includeEther ? [] : [nativeAsset.address])
     ]);
 
+    const listStyles = computed(() => {
```

Comment:
🤔 computed function will trigger here once, when `listElement` node is rendered?
may be it's more evident to calculate height in `onMounted` hook after `nextTick` promise settle

Reply:
![image](https://user-images.githubusercontent.com/20267733/174215438-42c6d26a-1cea-4ebb-a63f-8097e878459e.png)

`BalModal` does not insert the content of the modal at the moment it is `mounted`, because it uses `transition`. 

Also, we still have to use one reactive property, because setting styles on the element inside the script tag doesn't seem to be the best solution. 

In my opinion, this solution is quite suitable 👀